src/entries/popup/hooks/useSearchCurrencyLists.ts

```diff
@@ -508,6 +512,10 @@ export function useSearchCurrencyLists({
       return sections;
     }
 
+    if (popularAssets) {
```

Comment:
i think you need to make sure there is no overlap between assets in the popular section and assets in other sections. here's the app logic https://github.com/rainbow-me/rainbow/blob/develop/src/__swaps__/screens/Swap/hooks/useSearchCurrencyLists.ts

lmk if you want me to explain more